README.md

```diff
@@ -199,7 +199,7 @@ esptool.py --port /dev/ttyUSB0 --chip esp32 --before default_reset --after hard_
 Users report that [ESP_Flasher](https://github.com/Jason2866/ESP_Flasher/releases/) is suitable for flashing OpenDTU on Windows.
 
 #### Flash with [ESP_Flasher](https://espressif.github.io/esptool-js/) - web version
-It is also possible to flash it via the web tools which might be more convenient and is platformindependent.
+It is also possible to flash it via the web tools which might be more convenient and is platform agnostic.
```

Comment:
What do you mean with agnostic in this context? platform independent (ok, have to add a space) is a very common phrase.

Reply:
It basically means the same https://english.stackexchange.com/a/12903 ... changed it back anyway, was just opinion based